app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/event_target_parser.rb

```diff
@@ -9,6 +9,25 @@ def parse
     targets = []
 
     case ems_event[:event_type]
+    when /^image\.delete/
+      targets << InventoryRefresh::Target.new(
+        :association => :miq_templates,
+        :manager_ref => {:ems_ref => ems_event[:vm_ems_ref]},
```

Comment:
It is minior, but prefer to use the accessor methods like `ems_event.vm_ems_ref` so it is more obvious that this isn't just a hash